beta/nncf/tensorflow/quantization/initializers/minmax.py

```diff
@@ -164,7 +164,7 @@ def call(self, model, dataset=None, loss=None):
                             op.enabled = False
                             op_statistics.append((layer, op_name, op, minmax))
 
-        for step, (x, _) in enumerate(dataset):
+        for step, (x, _) in enumerate(self.dataset):
             if step >= self.num_steps:
                 break
```

Comment:
I suggest using [itertools.islice()](https://docs.python.org/3/library/itertools.html#itertools.islice). This is optional, at your discretion.
Motivation:
```python
for x, _ in islice(self.dataset, self.num_steps):
    model(x, training=False)
```
instead of
```python
for step, (x, _) in enumerate(self.dataset):
    if step >= self.num_steps:
        break
    model(x, training=False)
```

Reply:
Done